modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateServiceTest.java

```diff
@@ -152,7 +156,14 @@ private void _testAddNotificationTemplate(User user) throws Exception {
 
 			notificationTemplate =
 				_notificationTemplateService.addNotificationTemplate(
-					NotificationTemplateUtil.createNotificationContext(user));
+					NotificationTemplateUtil.createNotificationContext(
+						user, RandomTestUtil.randomString(), null,
```

Comment:
Could you update the `NotificationTemplateUtil.createNotificationContext(user)` to create a valid `NotificationRecipientSetting`, please?
So we can simplify this code using just the `NotificationTemplateUtil.createNotificationContext(user)`